src/__tests__/verify.test.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import Verify from '../App/Pages/verifyUser.page';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import thunk from 'redux-thunk';
+import '@babel/polyfill';
```

Comment:
you can add this in `package.json` so that everyone won't need to import this in every file.
under jest configuration
```
"setupFilesAfterEnv": [
			"./node_modules/jest-enzyme/lib/index.js",
			"@babel/polyfill"
		],
```